package.json

```diff
@@ -201,7 +201,7 @@
     "request@^2.83.0": "patch:request@npm%3A2.88.2#./.yarn/patches/request-npm-2.88.2-f4a57c72c4.patch",
     "request@^2.88.2": "patch:request@npm%3A2.88.2#./.yarn/patches/request-npm-2.88.2-f4a57c72c4.patch",
     "request@^2.85.0": "patch:request@npm%3A2.88.2#./.yarn/patches/request-npm-2.88.2-f4a57c72c4.patch",
-    "lavamoat-core@^14.2.0": "patch:lavamoat-core@npm%3A14.2.0#./.yarn/patches/lavamoat-core-npm-14.2.0-c453f4f755.patch",
```

Comment:
What is `lavamoat-core@npm`? Changing the lavamoat namespace without explanation looks suspicious.

Reply:
I believe this is something specific to how yarn 4 works. You'll notice that the deleted code is `"lavamoat-core@^14.2.0": "**patch:lavamoat-core@npm**%3A14.2.0#./.yarn/patches/lavamoat-core-npm-14.2.0-c453f4f755.patch",`, so `lavamoat-core@npm` was used before, even though the `@npm` was not in the key name.

We can get a more precise answer than this though...

Reply:
actually, I don't really know what I am talking about...

Reply:
This was done automatically when running `yarn patch`.

Reply:
Some comments in slack in response to this thread:

> - I would assume it might be a new default format in Yarn 4
> - Just patched it locally and saw the same format
> - Could also be a format that has existed pre Yarn 4 but is only used in some cases :shrugg:

> - The @npm syntax has been used since at least yarn3 in yarn.lock, yarn cli and in patch values to denote the source of the package.
> - This appears to be the first resolution added by yarn patch since upgrading to yarn 4.
> - Even in yarn 3 when you ran yarn patch it would ask for you to specify which version in the cases where there were multiple versions, and when you did that it would include the source @npm